src/app/shared/components/template/components/task-card/task-card.component.scss

```diff
@@ -69,11 +73,11 @@
     }
   }
 
-  &.background-secondary {
+  &[data-variant~="background-secondary"] {
     background-color: var(--ion-color-secondary-200);
   }
 
-  &.background-primary {
+  &[data-variant~="background-primary"] {
```

Comment:
nit(non-blocking) - could this now be a global style applied to any selector? E.g. `*[data-variant~="background-primary"]`... Although possibly less important if we think we're more likely to handle this through styles instead of variants

Reply:
I've enacted this. I did also consider changing the variant name to "background-primary-200", but I think that might imply a control over styles that doesn't exist yet (e.g. choosing other colour values) – whilst this is still a component _variant_, one of the handful that are available for the task card, "background-primary" makes sense to me as a name.

Reply:
Makes sense, will just need to be careful in the future if we do decide to deprecate/migrate the syntax for this case